lib/cosi/cosi.go

```diff
@@ -104,6 +104,14 @@ type Signature struct {
 	Response  abstract.Secret
 }
 
+// NewSignature returns a pre-initialized signature with Secret.One
+func NewSignature(s abstract.Suite) *Signature {
```

Comment:
It's used in the skipchain:

https://github.com/dedis/cothority/pull/403/files#diff-8cab4b17d6c04722d8a04ae4a1068cc6R112

I send the skipblocks around before signing them. But I can also move `NewSignature` to `skipchain/struct.go`